core/CronArchive.php

```diff
@@ -1742,20 +1738,19 @@ private function getUrlsWithSegment($idSite, $period, $date)
             $urlWithSegment = $this->getVisitsRequestUrl($idSite, $period, $dateParamForSegment, $segment);
             $urlWithSegment = $this->makeRequestUrl($urlWithSegment);
 
-            $periodInProgress = $this->isAlreadyArchivingAnyLowerOrThisPeriod($idSite, $period);
-            if ($periodInProgress) {
-                $this->logger->info("- skipping segment archiving for period '{period}' with segment '{segment}' because processing the period '{periodcheck}' is already in progress.", array('segment' => $segment, 'period' => $period, 'periodcheck' => $periodInProgress));
-                continue;
-            }
-
-            if ($cliMulti->isCommandAlreadyRunning($urlWithSegment)) {
-                $this->logger->info("- skipping segment archiving for '{segment}' because such a process is already in progress.", array('segment' => $segment));
+            if ($this->isAlreadyArchivingSegment($urlWithSegment, $idSite, $period, $segment)) {
```

Comment:
In theory we wouldn't need this check and could rely on the one in `->before()` but figured it might be good to have.